docs/snakefiles/rules.rst

```diff
@@ -1662,6 +1710,10 @@ It is possible to combine explicit group definition as above with pipe outputs.
 Thereby, pipe jobs can live within, or (automatically) extend existing groups.
 However, the two jobs connected by a pipe may not exist in conflicting groups.
 
+As with other groups, Snakemake will automatically calculate the required resources for the group job (see :ref:`resources <snakefiles-resources>`.
+The way resources are combined differs, though.
+Most resources, including ``runtime``, are summed accross all jobs involved in the pipe.
+Threads, however, is set by the job requesting the highest number of threads (``max()``).
```

Comment:
Sorry, now I am a bit confused (maybe we discussed this before, sorry). Why this behavior? Say A pipes into B, shouldn't the runtime of both still be the maximum of the two? Also, for the threads, don't we need the sum of their threads?

Reply:
Truthfully, I don't remember exactly why I set it up this way. I've never used the piping function myself; I know I was unsure how to do it, and I was hoping for some guidance.

I'll go ahead and change it to as you say. That makes things much simpler anyway: pipe jobs will behave like any other job.